cocos/renderer/core/gfx/GFXQueue.h

```diff
@@ -13,7 +13,7 @@ class CC_CORE_API GFXQueue : public GFXObject {
 public:
   virtual bool initialize(const GFXQueueInfo& info) = 0;
   virtual void destroy() = 0;
-  virtual void submit(const std::vector<GFXCommandBuffer*>& cmd_buffs) = 0;
+  virtual void submit(const std::vector<GFXCommandBuffer*>& cmd_buffs, GFXFence* fence = nullptr) = 0;
```

Comment:
It is better not use default parameter as possible.

Reply:
would function overloads be better for this purpose?

Reply:
Yep, i think it is better here.